src_sycl/CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ if(BACKEND STREQUAL "COMPUTECPP")
 
 	set(CMAKE_CXX_COMPILER ${ComputeCpp_DEVICE_COMPILER_EXECUTABLE})
 
+	set_property(TARGET ${BINARY_NAME} APPEND
+        PROPERTY COMPILE_OPTIONS
+		-std=c++17 -sycl-std=2020)
```

Comment:
you can set the properties `CXX_STANDARD 17` and `CXX_STANDARD_REQUIRED` to add the correct C++ standard flags. I will soon add support for setting the SYCL standard in a similar fashion.